util/log.py

```diff
@@ -13,6 +13,9 @@
 
 # https://stackoverflow.com/questions/25194864/python-logging-time-since-start-of-program/25196134#25196134
 class ElapsedFormatter:
```

Comment:
is `ElapsedFormatter` still an appropriate name then?

Reply:
not really... any ideas?

Reply:
`DialsLogfileFormatter`?